libs/sdk-ui-filters/src/AttributeFIlter@next/AttributeDropdown/types.ts

```diff
@@ -0,0 +1,30 @@
+// (C) 2019-2022 GoodData Corporation
+import { IPagedResource } from "@gooddata/sdk-backend-spi";
+import { IAttributeElement } from "@gooddata/sdk-model";
+
+export const emptyListItem: EmptyListItem = { empty: true };
+
+/**
+ * @public
```

Comment:
`@internal` (same for other interfaces in this file)

Reply:
fixed